ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -1,137 +1,133 @@
 ---
-title: Improve network performance with Route Reflectors
-description: Learn how to deploy Calico Route Reflectors to improve performance
-  of Kubernetes networking
+title: Improve network performance with route reflectors
+description: Learn how to deploy Calico route reflectors to improve the performance of Kubernetes networking.
 keywords: cluster, node, label, certificate, SAN
 ---
 
 >{% include enterprise_label_shortform.md %}
 
-UCP uses Calico as the default Kubernetes networking solution. Calico is
-configured to create a BGP mesh between all nodes in the cluster.
+Docker Universal Control Plane (UCP) uses [Calico](https://docs.projectcalico.org/v3.11/introduction/) as the default Kubernetes networking solution. Calico is configured to create a Border Gateway Protocol (BGP) mesh between all nodes in the cluster.
 
 As you add more nodes to the cluster, networking performance starts decreasing.
 If your cluster has more than 100 nodes, you should reconfigure Calico to use
```

Comment:
A suggestion for a smoother transition: Start this  sentence with "In fact, if your cluster...".